data/json/overmap/overmap_connections.json

```
@@ -9,7 +9,8 @@
       { "terrain": "road", "locations": [ "forest_without_trail" ], "basic_cost": 20 },
       { "terrain": "road", "locations": [ "forest_trail" ], "basic_cost": 25 },
       { "terrain": "road", "locations": [ "swamp" ], "basic_cost": 40 },
-      { "terrain": "bridge", "locations": [ "water" ], "basic_cost": 120 }
+      { "terrain": "dam", "locations": [ "water" ], "basic_cost": 120, "weight": 2 },
+      { "terrain": "bridge", "locations": [ "water" ], "basic_cost": 120, "weight": 1 }
```

```suggestion
      { "terrain": "dam", "locations": [ "water" ], "basic_cost": 120, "weight": 1 },
      { "terrain": "bridge", "locations": [ "water" ], "basic_cost": 120, "weight": 10 }
```
needs to be reverted after testing is done, feels weird there's so many dams